packages-exp/auth-exp/src/model/user.d.ts

```diff
@@ -32,8 +37,9 @@ export interface User extends externs.User {
   refreshToken: string;
   emailVerified: boolean;
   tenantId: string | null;
-  providerData: externs.UserInfo[];
+  providerData: ModifiableUserInfo[];
```

Comment:
👍 but this should probably be called MutableUserInfo

Reply:
Done